code/modules/ruins/spaceruin_code/forgottenship.dm

```diff
@@ -0,0 +1,115 @@
+// forgottenship ruin
+GLOBAL_VAR_INIT(fscpassword, generate_password())
+
+/proc/generate_password()
+	return "[pick(GLOB.phonetic_alphabet)] [rand(1,1000)]"
```

Comment:
```suggestion
	return "[pick(GLOB.phonetic_alphabet)] [rand(1000,9999)]"
```

Reply:
This might sound better, I guess, but I don't remember if, for example, PDA Uplinks had passcode numbers longer than 1000.